scripts/README.md

```diff
@@ -0,0 +1,104 @@
+# Flight sync/build/release scripts
+
+A pipeline for building the Flight iconset into a set of packages that can be consumed by other tooling and products.
+
+---
+
+## Usage
+
+The pipeline can be run manually, on your local machine, or via GitHub actions. 
+
+There are three distinct steps in the process:
+
+### Sync
+
+The synchronization step is executed by the `/scripts/sync.ts` file. To run this script use the following command in your CLI:
+
+```bash
+yarn sync
+```
+
+This action will:
+
+* Retrieve the assets metadata from Figma via REST API
+* Export the assets as `.svg` files into `/src/flight-icons-original/svg`
```

Comment:
question (non-blocking): 

i had tried to have the go export simply dump the files into https://github.com/hashicorp/flight/tree/main/flight-icons/icons (we could rename that directory to `dist` or `svg` or `src`, no opinion) but i couldn't get the exporter to traverse the tree, maybe it was broken though

maybe we can explore that after this and update the instructions of all the manual munging separately https://github.com/hashicorp/flight/blob/main/CONTRIBUTING.md#figma-export-go-instructions

Reply:
as far as I understand, the go exporter is expecting a very specific node/nesting structure, which in reality can be changed. this script doesn't care of where the icons are in the tree, because is using the "export" provided by the REST API directly.

Re. the readme, if you see in the /scripts/README you can see the new/updated instructions of how to run the "sync" script.

Reply:
sounds good, let's keep it as-is for now!